docs/books/computer-games/computer-battle-games/escape/index.rst

```diff
@@ -1,11 +1,26 @@
-=======
+=============
 Escape!
-=======
+=============
```

Comment:
As a stylistic thing, the underlining should match the length of the line being underlined. Most ReST compilers won't complain, but some do.